### ISSUE TITLE
Zone Air System Sensible Heating (or Cooling) Energy should describe types of equipment included

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4291]
Per MJW comment, took example file that had AirLoop and ZoneHVAC Equipment and added output variables Zone Air System Sensible Heating Energy and Zone Air System Sensible Cooling Energy and ran with v8.1.  Don't know what MJW comment means that says "add output variables and see if they line up".  Don't know what "line up" means here?  

2/19/2014 - MJW - What I meant was, if the zone has baseboard heat, for example, is the heating output from the baseboard included in Zone Air System Sensible Heating Energy?  Essentially, we need to test every type of zone equipment to see if it's included or not and make a list.  Given the new name, it may be as simple as saying that only forced-air units are included and all other HVAC equipment types (baseboard, low-temp radiant, and high-temp radiant) are not included.

RHH Response 3/4/2014
Looked at only the 5 baseboard heating options so far and YES, the baseboard heating is showing up in the Zone Air System Sensible Heating Energy output variable, so need to update I/O Ref to indicate that.  I tested these systems by setting the air system availablity schedule to 0.0.  Have not tested other types of zone equipment yet.

Helpdesk Ticket 2049
Example files
DOASDualDuctSchool.idf
5ZoneElectricBaseboard.idf
5ZoneSteamBaseboard.idf
BaseBoardElectric.idf
5ZoneWaterCooled_Baseboard.idf

**Comment by smldis** [comment]
I found this issue and it seems related to my problem:
I am trying to get energy consumption by zone. Will I be able to match the sum over zones of `Zone Air System Sensible Heating Energy` to the overall consumption related to: `End uses -> Natural Gas` (taking into account efficiency etc)?
The model has a centralized boiler and one radiator for each room of type `ZoneHVAC:Baseboard:RadiantConvective:Water`

**Comment by RKStrand** [comment]
@smldis To answer your question, I think you should be able to see a correlation between the Zone Air System Sensible Heating Energy and the natural gas use given that you are using a natural gas boiler and you are getting heating from a ZoneHVAC:Baseboard unit.  As RHH reported above and as I have at least verified with the code but will also double-check as part of addressing this issue, the baseboard output is included in the Zone Air System variable.

**Comment by smldis** [comment]
@RKStrand 
I tried and there is a correlation:
![immagine](https://user-images.githubusercontent.com/682767/61377863-570bba00-a8a4-11e9-9789-2ff718ca822a.png)
There is a 1.35 factor to match the values. What's happening?

**Comment by mjwitte** [comment]
@smldis Output from the baseboard should equal boiler heating energy plus Pump Fluid Heat Gain Energy.

**Comment by smldis** [comment]
@mjwitte It's not, I checked `Pump Fluid Heat Gain Energy`. It is negligible in my case. Could you point me to the relevant source code files?

**Comment by RKStrand** [comment]
@smldis In looking at your file and after hearing that there is a constant factor increase between the zone system load and the boiler load and also seeing that you are using a radiant-convective baseboard, I wonder if the explanation is in the fact that some of the energy from the baseboard is actually radiant energy.  I think that the load you are seeing is the convective output of the baseboard.  But the input to the baseboard will be higher because it includes both the convective and radiant portions.  Check the baseboard input or post your baseboard description here and see if this explains the discrepancy.

@mjwitte The problem that @smldis is seeing is not related to the original problem that you posted so a review of the fix for that issue can be be done at any time.

**Comment by smldis** [comment]
@RKStrand What you are saying makes sense now. I tried to search for the variables that would allow to prove this, I obtained this result:
![immagine](https://user-images.githubusercontent.com/682767/61880262-db89b880-aef4-11e9-84e3-da34e76129e5.png)
There is still a small difference ~0.1%. If the result is correct, this should be documented because having energy consumptions by zone or radiator is very important.

My radiators are all similar to this one:
```
ZoneHVAC:Baseboard:RadiantConvective:Water,
    5210 Water Radiator,     !- Name
    2,                       !- Availability Schedule Name
    5210 Water Radiator Hot Water Inlet Node,  !- Inlet Node Name
    5210 Water Radiator Hot Water Outlet Node,  !- Outlet Node Name
    75.000,                  !- Rated Average Water Temperature {C}
    0.063000,                !- Rated Water Mass Flow Rate {kg/s}
    HeatingDesignCapacity,   !- Heating Design Capacity Method
    autosize,                !- Heating Design Capacity {W}
    ,                        !- Heating Design Capacity Per Floor Area {W/m2}
    ,                        !- Fraction of Autosized Heating Design Capacity
    autosize,                !- Maximum Water Flow Rate {m3/s}
    0.0100,                  !- Convergence Tolerance
    0.300,                   !- Fraction Radiant
    0.100,                   !- Fraction of Radiant Energy Incident on People
    5210_Wall_4_0_0,         !- Surface 1 Name
    0.0007,                  !- Fraction of Radiant Energy to Surface 1
...
...
```
I would kindly ask if someone can give me a confirmation on the correct choice of the output variables.
Would it be possible to have a reference on the source code and documentation about those?

Thank you very much

**Comment by RKStrand** [comment]
@smldis Yes, those are the correct variables.  Code for the various radiant-convective baseboard models can be found in: ElectricBaseboardRadiator.cc, HWBaseboardRadiator.cc, and SteamBaseboardRadiator.cc.  Documentation on the baseboards themselves are in both the Input/Output and the Engineering References.

As for adding to the documentation, I will go ahead and add some text to the three radiant-convective models to let users know how to get the total heating gain from the baseboard (essentially do what you did--add the convective and radiant gains).

**Comment by smldis** [comment]
@RKStrand also, the original title of this thread is nonsense: 'Zone Air System Sensible Heating (or Cooling) Energy' is not about only about the equipments but also about the type of heat transfer, I would propose (maybe naively) to deprecate all the 'Zone Air System Sensible' stuff and introduce a good variable for the zones that is the sum of all the contributions of the radiators of the zone.

**Comment by RKStrand** [comment]
@smldis Well, that's a much bigger discussion that perhaps should have a separate issue logged and a lot more discussion.  I understand what you are saying to some extent.  There are a lot of radiant components that are not included in this variable, but there has not been any discussion of wether or now to resolve that.  It's not a trivial task unfortunately.

**Comment by mjwitte** [comment]
I don't understand why one would want to eliminate "Zone Air System Sensible ...". It provides a useful summation of all of the convective heat transfer to the zone. Adding a summation of radiant HVAC equipment output would be useful.

**Comment by RKStrand** [comment]
@smldis @mjwitte Ok, I have made some changes to the baseboard I/O docs to clarify things here.  This should show up in the latest commit that I just made.

I agree with Mike that the addition of the radiant HVAC equipment output might be useful.  That definitely needs to be a separate issue.  I'm not sure, but that could be a bigger ask than anticipated.  Without looking at the code, I suspect that it will not be a trivial task because it will have to include a lot of different equipment types.  If this is something that might be useful, I'd ask that one of you start a new issue and tag me as the assignee.  I'm the most likely candidate to work on this, but I think it will be better if the request comes from someone else.  That way, it doesn't look like I'm just creating work for myself. :)  Thanks!

**Comment by mjwitte** [comment]
@RKStrand It might be good to add some text to the description of this output variable to explain what types of HVAC input get included.

**Comment by RKStrand** [comment]
@mjwitte Yup, agreed.  That was the original intent and was the first thing that I added as part of this work.  See the new input-output reference that gets built for this pull request and you should see new text that specifically declares what systems will contribute to this.  The issue that @smldis brought up was a slight tangent perhaps, but I included doc changes for that here as well.  The only thing that needs to be split off into a new issue is the concept of a Zone Air System Radiant Heating/Cooling Rate/Energy variables.  Unless I am misunderstanding or missing something, I think this work is ready for review.

**Comment by smldis** [comment]
Good to go, I previously misunderstood the terms in Zone Air System Sensible Heating.
For 'Zone Air System Radiant Heating/Cooling Rate/Energy' I am not sure the naming is ok since the majority of the heat is not going to the Zone Air. Maybe 'Zone System {Radiant} Heating/Cooling ...'?

**Comment by mjwitte** [comment]
@RKStrand Sorry - I was looking at just today's commit rather than the full pull request. I see it's there now in #7392.

**Comment by mjwitte** [comment]
Closed via #7392
